### ISSUE TITLE
Translation of admonition text

### ISSUE THREAD

**Issue by eventhorizonsky** [TwnKey/YCTT#3]
Hello, may I ask if you are familiar with how to translate text like the one shown in the image? It appears that this particular text is not included in the output provided by YsIIScenaTool
![3782dfe970c393b0ced4275cecc77bc5_720](https://github.com/user-attachments/assets/515549ff-b4a0-4678-9f70-644a6d43fe42)

**Comment by TwnKey** [comment]
🤔I don't know where it comes from. Maybe its a picture? Or in the exe? I might check this weekend

**Comment by TwnKey** [comment]
I had a look. This one specifically is in C_OLF00.SKI, I wrote a tool to extract the pictures as PNG and see. ![frame_0](https://github.com/user-attachments/assets/76b5d256-3b9a-4a7a-a84a-f5d01647b3d7)
https://github.com/TwnKey/YCTT/releases/download/v1.0/ImageConverter.exe

But it only extracts for now. And with the wrong palette (the real palette is a data from the .exe file, there is no way to guess it from the picture only).

I have a rough idea of what I need to do to reinsert an edited file but it might take some time. I am planning to work slowly on it from time to time, so hopefully you don't need it right now. If you end up finishing everything in your translation, if I'm not done with this tool by then, just wake me up here and I'll finish asap.

NOTE TO MYSELF: Check https://github.com/TwnKey/YCTT/blob/main/ImageConverter/Strat%C3%A9gie%20pour%20la%20suite.txt

**Comment by eventhorizonsky** [comment]
Thanks a lot! we are not in a hurry. Take your time please, and if possible, we would appreciate it if you could help us look into replacing the opening animation video in the game. Some members of our team wanted to replace it with a higher quality version.

**Comment by TwnKey** [comment]
Your team has a QQ? Where do you communicate?

**Comment by eventhorizonsky** [comment]
Yes, we communicate on QQ, and the group number is 978408928.Feel free to join us anytime if you are interested.

**Comment by eventhorizonsky** [comment]
ImageConverter has now stabilized and resolved this issue.
